tests/MageTest/Task/FactoryTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MageTest\Task;
+
+use Mage\Task\Factory;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @group Mage_Task
+ * @coversDefaultClass Mage\Task\Factory
+ */
+class FactoryTest extends PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        require_once __DIR__ . '/MyTaskExample.php';
+    }
+
+    /**
+     * @covers Mage\Task\Factory
+     */
+    public function testLoadFromExternalNamespace()
+    {
+        $factory = new Factory;
+
+        $configMock = $this
+            ->getMockBuilder('Mage\Config')
```

Comment:
You don't need to instantionate `MockBuider` here. It'll be ok to do `$this->getMock('Mage\Config')`